app/styles/app.scss

```diff
@@ -556,12 +558,13 @@ hr {
             opacity: 1;
         }
     }
-    .row {
-        margin: 0;
-    }
-    label, input, textarea {
-        display: block;
-    }
+    // THESE STYLES ARE INTERFERING WITH MY STYLES. 
```

Comment:
I wouldn't say these styles are general _across the app_, as they are nested under `.preprint-form-section`. Though I agree they could be more specific.